docs/working-with-projects/working-with-images/create-tag-retention-rules.md

```diff
@@ -15,58 +15,68 @@ A tag retention rule has 3 filters that are applied sequentially, as described i
 |---|---|---|
 |First|Repository or repositories|Identify the repository or repositories on which to apply the rule. You can identify repositories that either have a certain name or name fragment, or that do not have that name or name fragment. Wild cards (for example `*repo`, `repo*`, and `**`) are permitted. The repository filter is applied first to mark the repositories to which to apply the retention rule. The identified repositories are earmarked for further matching based on the tag criteria. No action is taken on the nonspecified repositories at this stage.|
 |Second|Quantity to retain|Set which tags to retain either by specifying a maximum number of tags, or by specifying a maximum period for which to retain tags.|
-|Third|Tags to retain|Identify the tag or tags on which to apply the rule. You can identify tags that either have a certain name or name fragment, or that do not have that name or name fragment. Wild cards (for example `*tag`, `tag*`, and `**`) are permitted.|
+|Third|Tags to retain|Identify the tag or tags on which to apply the rule. You can identify tags that either have a certain name or name fragment, or that do not have that name or name fragment. Wild cards (for example `*tag`, `tag*`, and `**`) are permitted. And there's a checkbox to determine whether **untagged artifacts** should also be considered.|
```

Comment:
-> 'Identity the tags on which to apply the rule'

Reply:
-> 'Use the checkbox to select whether untagged images should be captured as part of the set of images eligible for tag retention'